scripts/hid_configurator/gui/gui.py

```diff
@@ -142,13 +172,31 @@ def dfu(self, update_progressbar):
         else:
             info = 'DFU transfer failed'
             message = 'Restart application and try again'
-        print(info)
+
         dfu_label.text = info
         notification.notify(app_name='GUI Configurator', app_icon='nordic.ico', title=info, message=message)
 
-    def perform_fwreboot(self):
-        print('Performing fwreboot, please wait')
-        self.device.perform_fwreboot()
+    def dfu_fwreboot(self, update_reboot_animation):
+        dfu_label = self.root.ids.dfu_label
+        progression_label = self.root.ids.dfu_buttons.ids.progression_label
```

Comment:
created variable just to make single assignment at the end of the function. Please use self.root.ids.dfu_buttons.ids.progression_label.

Reply:
done